### DISCUSSION TITLE
GraphQl RootResolvers are now operating on sync thread?

### DISCUSSION THREAD

**Discussion by tara-9** [graphql-java-kickstart/graphql-spring-boot#858] [Q&A]
```
public class AsyncServletProperties {

  public static final Duration DEFAULT_TIMEOUT = Duration.ofSeconds(30);

  private boolean enabled = true;
```

It was on graphql 12. Now in 13  private boolean enabled = false. What will be the correct value going forward? 
@oliemansm

**Comment by oliemansm** [comment]
As stated in the release notes the default value was inverted to false. You can set it explicitly to true if you want the behavior of v12 of course.
